examples/pihole/docker-compose.yml

```
@@ -31,8 +31,7 @@ services:
       - "TZ=Europe/Paris"
       - "PROXY_LOCATION=pihole"
       - "VIRTUAL_PORT=80"
-      - "DNS1=172.20.0.3#5053"
-      - "DNS2=1.1.1.1"
+      - "PIHOLE_DNS=172.20.0.3#5053;1.1.1.1"
```

As stipulated in https://github.com/pi-hole/docker-pi-hole/pull/727#issue-769290037, it should be:

```suggestion
      - "PIHOLE_DNS_=172.20.0.3#5053;1.1.1.1"
```

---

@crazy-max Good catch! Somehow the underscore got missing. Changed it.